### PR TITLE
Use ubuntu as ssh user for kops canaries

### PR DESCRIPTION
**Pull request by johngmyers**
/cc @rifelpet

### CODE REVIEW

File: config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml [hunk 1]
```diff
@@ -244,6 +245,8 @@ periodics:
       args:
       - --cluster=e2e-kops-gce-canary.k8s.local
       - --deployment=kops
+      - --kops-ssh-user=ubuntu
+      - --env=KUBE_SSH_USER=ubuntu
```
Comment (rifelpet) [inline config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml:249]: I'm not sure this should be set on the GCE jobs (see https://github.com/kubernetes/test-infra/pull/17950#issuecomment-644726859 )

/lgtm cancel

**Comment by johngmyers** [comment]
@wojtek-t Anything else you need to approve this PR?

**Comment by BenTheElder** [comment]
/approve
But please move this to being owned by the kops team.
We're freezing kubetest and not actively working on these.

**Pull request merged**